dht/i_partitioner.hh

```diff
@@ -264,14 +264,6 @@ public:
      */
     virtual token midpoint(const token& left, const token& right) const = 0;
 
-    /**
-     * @return A token smaller than all others in the range that is being partitioned.
-     * Not legal to assign to a node or key.  (But legal to use in range scans.)
-     */
-    token get_minimum_token() {
-        return dht::minimum_token();
-    }
-
     /**
      * @return a token that can be used to route a given key
```

Comment:
You can squash all those remove unused into one patch.

Reply:
done.